text/1589-rustc-bug-fix-procedure.md

```diff
@@ -98,29 +98,31 @@ What follows is a template for tracking issues.
 
 ---------------------------------------------------------------------------
 
-This is the **summary issue** for the `YOUR_LINT_NAME_HERE`
-future-compatibility warning and other related errors. The goal of
-this page is describe why this change was made and how you can fix
-code that is affected by it. It also provides a place to ask questions
-or register a complaint if you feel the change should not be made. For
-more information on the policy around future-compatibility warnings,
-see our [breaking change policy guidelines][guidelines].
-
-[guidelines]: LINK_TO_THIS_RFC
-
-#### What is the warning for?
-
-*Describe the conditions that trigger the warning and how they can be
-fixed. Also explain why the change was made.**
-
-#### When will this warning become a hard error?
-                    
-At the beginning of each 6-week release cycle, the Rust compiler team
-will review the set of outstanding future compatibility warnings and
-nominate some of them for **Final Comment Period**. Toward the end of
-the cycle, we will review any comments and make a final determination
-whether to convert the warning into a hard error or remove it
-entirely.
+#### What is this lint about
+
+*Describe here what kinds of Rust code will cause the lint to trigger.*
+
+*Motivational text or historical information is also of use, but the
+most important thing is to explain what kind of code the lint is
+detecting.*
+
+*You may wish to include a link to this RFC as part of explaining the
+lint in some manner, for example:*
+
+For more information on the policy around future-compatibility
+warnings, see our [breaking change policy guidelines][RFC 1589].
+
+[RFC 1589]: https://github.com/rust-lang/rfcs/blob/master/text/1589-rustc-bug-fix-procedure.md
+
+#### How to fix this warning/error
+
+*Explain here what the developer needs to do to address the warning.*
+
+#### Current status
```

Comment:
On the other hand, this seems like an improvement, but perhaps a bit premature -- I'm not sure if we always take these three steps? Still, I like the idea of showing the process more graphically, and I think we should adopt a "standard" time frame that is tighter than the "open-ended process" suggested by the old template.

Reply:
We probably don't always take these steps but I think we often do and it seems like a good procedure :) -- I agree with standardizing the time-frame, or at the very least having a time-frame at all.

Reply:
So is there actually something for me to do here? I think starting with the provided three steps in the template makes more sense than leaving them out. 

Do you want me to add an explicit note saying that people have the option to revise that section on a case-by-case basis?